.github/workflows/ci.yml

```diff
@@ -5,10 +5,8 @@ on:
     branches: [master]
   pull_request:
     branches:
-    - '**'
+      - "**"
   workflow_dispatch:
-    branches:
```

Comment:
Why are we removing them?

Reply:
The workflow_dispatch trigger does not require or support a branches key. By default, it allows manual triggering regardless of branch.
https://docs.github.com/en/webhooks/webhook-events-and-payloads#workflow_dispatch